bokeh/models/renderers.py

```diff
@@ -145,6 +145,12 @@ def _check_bad_column_name(self):
     being hovered over by a HoverTool.
     """)
 
+    mute_glyph = Instance(Glyph, help="""
```

Comment:
I chose `mute` over `muted` because the later doesn't follow the convention. Otherwise we would have `selected_glyph`, `hovered_glyph`, etc.